DQMOffline/JetMET/src/JetAnalyzer.cc

```diff
@@ -1910,6 +2084,12 @@ void JetAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetu
     if(isMiniAODJet_ && (*patJets)[ijet].isPFJet()){
       pat::strbitset stringbitset=pfjetIDFunctor.getBitTemplate();
       jetpassid = pfjetIDFunctor((*patJets)[ijet],stringbitset);
+      if(fabs ((*patJets)[ijet].eta()>3.0)){
```

Comment:
The bracket of fabs needs to be closed after eta().
Would you mind putting this change directly in the pfjetIDFunctor to avoid this duplication?